manifests/init.pp

```diff
@@ -95,6 +95,16 @@
 #             }]
 #     An array of API endpoints to connect uchiwa to one or multiple sensu servers.
 #
+#  [*users*]
+#    Array of hashes
+#    An array of user credentials to access the uchiwa dashboard. If set, it takes
+#    precendence over 'user' and 'pass'.
+#    Example: [{
```

Comment:
Can you expand this example to show both a rw and ro user and use markdown to make it easier to read? Also eventually the keys will need to be quoted as strings for future-parser compat. Here I think copy/paste this:

```
# Example:
# ```
# [{
#   'username' => 'rouser1',
#   'password' => 'pass1',
#   'readonly'   => false,
# },
# {
#   'username' => 'rwuser1',
#   'password' => 'pass1',
#   'readonly  => false,
# }]
# ```
#
```